inst/extdata/emissions/A41.GWP.csv

```diff
@@ -0,0 +1 @@
+# File: A41.GWP.csv
# Title: Global warming potential values for fgases
# Units: tonnes
# Source: 2007 IPCC AR4 p212 
# Description: The global warming potential of Fgases over a 100-year time horizon
Gas,HFC_PFC,GWP
C2F6,PFC,12200
CF4,PFC,7390
HFC125,HFC,3500
HFC134a,HFC,1430
HFC143a,HFC,4470
HFC152a,HFC,124
HFC227ea,HFC,3220
HFC23,HFC,14800
HFC236fa,HFC,9810
HFC245fa,HFC,1030
HFC32,HFC,675
HFC365mfc,HFC,794
HFC43,HFC,1640
SF6,PFC,22800
```

Comment:
Not sure what to do about this.  The last row/column cell is missing in the old and new.  Should I alter both?